www/js/insert-slide.js

```diff
@@ -3,13 +3,28 @@ const electron = require('electron');
 
 const analytics = electron.remote.getGlobal('analytics');
 const tingle = require('./vendor/tingle');
-const strings = require('./strings');
+const strings = require('./strings.json');
```

Comment:
`setLangSliderVal` function needs to be updated as well. `strings.announcemenetPlaceholder` is undefined.